src/CommunityToolkit.Aspire.Hosting.EventStore/README.md

```diff
@@ -0,0 +1,37 @@
+# CommunityToolkit.Aspire.Hosting.EventStore library
+
+Provides extension methods and resource definitions for the .NET Aspire AppHost to support running [EventStore](https://www.eventstore.com) containers.
+
+## Getting Started
+
+### Install the package
+
+In your AppHost project, install the package using the following command:
+
+```dotnetcli
+dotnet add package CommunityToolkit.Aspire.Hosting.EventStore
+```
+
+### Example usage
+
+Then, in the _Program.cs_ file of `AppHost`, add a EventStore resource and consume the connection using the following methods:
```

Comment:
```suggestion
Then, in the _Program.cs_ file of app host, add a EventStore resource and consume the connection using the following methods:
```

Reply:
Addressed in the last commit: https://github.com/CommunityToolkit/Aspire/pull/277/commits/7deb7d5bc32aa830d8266342076c010509f75b88